Fix: review the partnership page

lwasser opened the pull request:
Enhancing this page with updated information. 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206583360767794
  - https://app.asana.com/0/0/1206650250496717

kierisi commented on images/peer-review-partners-process.png:
I don't think it's a priority, but at some point I think that we could clarify this image even further to make it more clear that the community standards are incorporated into the overall review through a customized review process.

lwasser replied:
that would be great! i am trying to create really simple clear graphics so people understand how this partnership works! if it can be more clear than i'm ✔️  💯  !!

lwasser replied:
i also just noticed that this is missing the JOSS piece. hmmm

kierisi commented on images/pyos-partnerships-peer-review.png:
the numbers are throwing me a bit because they're sequential, but visually it looks like JOSS publication and community affiliation happen at the same time. because of the arrows and how the image is set up, I'm wondering if we could just omit the numbers all together.

lwasser replied:
ahhhh.i see so you're reading the numbers as sequential. the way i pitch this is that you get 3 "wins" with a single review. so they are intended to be a "count" like win 1. win 2, win 3. i'm not sure how to make that more clear however but i see your point as well. there's no real order here. in theory affiliated and pyOpenSci will likely happen at the same time given how things are setup. JOSS does have to happen after. our review is the only core piece - the thing that everyone goes through. not everyone will go on to JOSS or become affiliated.

kierisi replied:
would it be helpful to split them out into two separate graphics? one with the arrows, and then a text-based graphic with something like:

3 benefits to partnership
1.
2.
3.

kierisi commented on partners/scientific-communities.md:
```diff
@@ -1,38 +1,47 @@
+
+<img src="../images/pyopensci-pillars-flower.png" alt="Flower" width="500">
+
+pyOpenSci has three core programs. Community partnerships is a program that allows domain specific scientific Python communities to leverage our peer review process rather than creating their own process. Through this partnership maintainers can become affiliated with both a domain specific community, published in the Journal of Open Source Software and also become a part if the pyOpenSci communi ...
```
```suggestion
pyOpenSci has three core programs: program one, program two, program three. Community partnerships allow domain-specific scientific Python communities to leverage the pyOpenSci peer review process rather than creating their own process. Through this partnership, maintainers can become affiliated with a domain-specific community, published in the Journal of Open Source Software (JOSS) and join the pyOpenSci community.
```

lwasser replied:
@kierisi your in place text change makes the text a bit confusing with program one, program two etc. are you suggesting that  i list the name of each program there? or we could add numbers before each section? i just wouldn't want generic text.

lwasser replied:
if you wanted me to insert the program names a nice approach could be to only edit the text inline where you are confident in actually changing the text (if i merge this the text will now say program one, program two, etc) . then add a note to the suggested change that says - i'd spell out the program names or something like that. because as is i can't merge this (unless i am mis understanding the intent that is!)

lwasser replied:
ok i have. just rewritten this all alt tag and caption so it's more clear. resolving!

kierisi commented on partners/scientific-communities.md:
```diff
@@ -1,38 +1,47 @@
+pyOpenSci has three core programs. Community partnerships is a program that allows domain specific scientific Python communities to leverage our peer review process rather than creating their own process. Through this partnership maintainers can become affiliated with both a domain specific community, published in the Journal of Open Source Software and also become a part if the pyOpenSci communi ...
+:::
+
+Numerous science domain-specific scientific Python communities are developing and maintaining their own peer review
```
```suggestion
Numerous domain-specific scientific Python communities are developing and maintaining their own peer review
```

kierisi commented on partners/scientific-communities.md:
```diff
@@ -1,38 +1,47 @@
+:::
+
+Numerous science domain-specific scientific Python communities are developing and maintaining their own peer review
+processes to track and recommend packages that relevant to their science domain.
```
```suggestion
processes to track and recommend packages relevant to their specific area(s) of expertise.
```

kierisi commented on partners/scientific-communities.md:
```diff
@@ -1,38 +1,47 @@
+processes to track and recommend packages that relevant to their science domain.
+However, running a high quality peer review process takes time and resources.
+
+pyOpenSci offers a program where communities can partner with us, and leverage our peer review process, which is
```
```suggestion
pyOpenSci offers a program where communities can partner with us and leverage our peer review process, which is
```

kierisi commented on partners/scientific-communities.md:
```diff
@@ -1,38 +1,47 @@
+However, running a high quality peer review process takes time and resources.
+
+pyOpenSci offers a program where communities can partner with us, and leverage our peer review process, which is
+built on community-driven packaging guidelines and standards. Through this partnership, we enhance our standard review process with a community-specific
```
```suggestion
built on community-driven packaging guidelines and standards. Through this partnership, we enhance our standard review process with a custom review process
```

lwasser replied:
i really appreciate the concept here but i'm slightly worried about the word "custom" which might make it sound like we can totally modify our process. if there is a better word to suggest adding a layer on top of an existing structure i think that would be better. Community specific related to adding guidelines specific to that community but the process is still the same (with that additional layer). i know this is very nuanced but i wouldn't want someone to think we can totally customize our review process as that would be too much work to maintain. i'll think about other words or perhaps you have ideas too!! i may merge this before you are back only because of astropy moving forward now BUT  we can also always go back in and modify!

lwasser replied:
ok i know where this is from now. and i'll think about what words we want to use. it's a custom step rather than a custom review process. maybe i'll adjust the graphic language and finesse this...

kierisi commented on partners/scientific-communities.md:
```diff
@@ -1,38 +1,47 @@
+
+pyOpenSci offers a program where communities can partner with us, and leverage our peer review process, which is
+built on community-driven packaging guidelines and standards. Through this partnership, we enhance our standard review process with a community-specific
 step that adds your specific requirements for package affiliation.
```
```suggestion
that integrates your specific requirements for package affiliation.
```

kierisi commented on partners/scientific-communities.md:
```diff
@@ -1,38 +1,47 @@
 
-Here we discuss our process for
-supporting this community affiliation partnership.
+Through this partnership we add an additional layer of checks to our peer review process that align with a domain specific communities needs. Often these checks are around packages using existing core functionality of community tools, and adhering to community specific data structure formats.
```
```suggestion
Through this partnership, we add an additional layer of checks to the pyOpenSci peer review process that align with a domain-specific community's needs. Often these checks are around packages using existing core functionality of community tools and adhering to community-specific data structure formats.
```

kierisi commented on partners/scientific-communities.md:
```diff
@@ -52,12 +61,15 @@ Our vibrant community is filled with experts in the Python packaging space. A
 partnership with us means that you will always have support if you run into
```
```suggestion
partnership with pyOpenSci means that you will always have support if you run into
```

kierisi replied:
can we be more specific on what "support" looks like? maybe something like "always have access to support from our community of Python experts" etc.

lwasser replied:
great feedback! i've modified it to this:


> Our vibrant community is filled with experts in the Python packaging space. A
> partnership with us means that you will always have support from our diverse community of Pythonistas if you run into
> Python packaging and software maintenance challenges.

kierisi commented on partners/scientific-communities.md:
```diff
@@ -52,12 +61,15 @@ Our vibrant community is filled with experts in the Python packaging space. A
+:::{admonition} What's in the works
+:class: note
+
+We are currently enhancing our peer review process and working together on:
```
```suggestion
We are currently enhancing our peer review process and collaborating on:
```

kierisi replied:
it was unclear who the "we" is in "working together"

lwasser replied:
totally - i've changed it as follows:

> - Adding an editorial bot to our peer review infrastructure to streamline our review processes.
> - Collecting package metrics to track health and also understand maintenance status.

kierisi commented on partners/scientific-communities.md:
```diff
@@ -52,12 +61,15 @@ Our vibrant community is filled with experts in the Python packaging space. A
+We are currently enhancing our peer review process and working together on:
 
-We are currently working on several items to improve our review process and followup activities with package maintainers including:.
+-Adding an editorial bot to our peer review infrastructure to further streamline our processes.
```
```suggestion
- Adding an editorial bot to our peer review infrastructure to further streamline our processes.
```

kierisi commented on partners/scientific-communities.md:
```diff
@@ -52,12 +61,15 @@ Our vibrant community is filled with experts in the Python packaging space. A
 
-We are currently working on several items to improve our review process and followup activities with package maintainers including:.
+-Adding an editorial bot to our peer review infrastructure to further streamline our processes.
+- Collecting metrics around package maintenance to track health and to help identify when packages become unmaintained.
```
```suggestion
- Collecting metrics around package maintenance to track package health and to help identify when packages become unmaintained.
```

kierisi replied:
both of these processes are important, but they sound kind of...sterile? slightly negative? in other words, if I'm interested in collaborating with pyOS, these make it sound like:
- pyOS doesn't have a fully worked-out process
- packages are unhealthy

the first one is fine, but I think rewording the second to be more positive, along the lines of "to track package health and ensure all pyOS packages are well-maintained"

kierisi replied:
I'm also wondering if there's anything else we could talk about here that would be a direct benefit to potential partners - something they'd see and go "oh, that's interesting!" or "we could really use that!"

lwasser replied:
A bit of context:

Every partner community i've talked to has a pain point of tracking these things! that is what they want. currently many have these manual badges that quickly get out of date. so us automating it and working together as a community is that "we could really use that" feature that everyone wants! the bot is more for us / review to ensure consistency. it's so easy to forget steps such as adding labels, etc.  

however - i agree that it could read as  unhealthy AND we don't have a process yet - part of the collab will be figuring out what these checks look like. 

We already did work together on our archiving policy in the peer review guide with a few communities. So the health metrics will be similar i think and will be a learning process! 

with that said i've rewritten as follows to try to address that "it sounds negative" vibe it was giving

> 
> pyOpenSci is enhancing our peer review process and working on:
> 
> - Adding an editorial bot to our peer review infrastructure to streamline our review processes.
> - Collecting package metrics to track health and also understand maintenance status.

we could consider using a different word than health. i'm just not sure what word to insert!

kierisi replied:
@lwasser I don't think there's an issue with the word health, but rather "help identify when packages become unmaintained" - so what I'm thinking is more "help identify when packages are in need of maintenance" or something to indicate that we're keeping an eye on the issue. and if we offer assistance on intervention (finding new maintainers, etc.) mentioning that as well!

kierisi reviewed (commented):
added suggestions for both content and grammar!

lwasser commented on partners/scientific-communities.md:
```diff
@@ -1,38 +1,47 @@
+:::{figure-md} pyopensci-joss-community
 
-<img src="../images/peer-review-partners-process.png" alt="Diagram showing the pyOpenSci peer review process. At the top there are 4 boxes labeled submit package for review, package review, package accepted and fast track in JOSS. Each of these are steps in our review process. At the bottom of the diagram are yellow boxes highlighting the fact that in the package review, an additional layer of re ...
+<img src="../images/pyos-partnerships-peer-review.png" alt="" width="400">
```
NOTE: missing an alt tag for the new graphic!!

lwasser replied:
added!

lwasser commented on partners/scientific-communities.md:
```diff
@@ -1,38 +1,47 @@
-- Be accepted by pyOpenSci.
-- Undergo a fast-track review and get published with JOSS (if in scope).
-- Achieve affiliation with your domain-specific community.
+<img src="../images/pyopensci-pillars-flower.png" alt="Flower" width="500">
```
fixed alt tag!

lwasser commented on partners/scientific-communities.md:
```diff
@@ -1,38 +1,47 @@
+
+<img src="../images/peer-review-partners-process.png" alt="." width="500">
+
+Through this partnership, we add an additional layer of checks to the pyOpenSci peer review process that align with a domain-specific community's needs. Often these checks are around packages using existing core functionality of community tools and adhering to community-specific data structure formats.
```
Ahhh @kierisi i see where that word customized came from. i used it in this graphic. i'll revisit that word as it's bothering me now. maybe customized review step? rather than the entire review being custom?

lwasser commented:
ok! i think i've addressed everything here now. and i've updated /added alt tags and such. i'll give this one more review with fresh eyes tomorrow and will merge. and we can always make small pr updates to things as we discuss language! thank you for the review @kierisi 🚀

lwasser commented:
Ok. i am going to merge this now. it's in a good enough state that i think we can merge and then open a new PR to finesse as needed! many thanks for all of the feedback @kierisi !!
